requirements-dev.txt

```diff
@@ -1,5 +0,0 @@
-tox
-pylint
-pytest
-mock
-sphinx
```

Comment:
the only modules needed to be install are sphinx and tox and they are included in the `install section`.